asv_bench/benchmarks/package.py

```diff
@@ -0,0 +1,22 @@
+"""
+Benchmarks for pandas at the package-level.
+"""
+import subprocess
+import sys
+
+from pandas.compat import PY37
+
+
+class TimeImport:
+    def time_import(self):
+        if PY37:
+            cmd = [sys.executable, "-X", "importtime", "-c", "import pandas as pd"]
+            p = subprocess.run(cmd, stderr=subprocess.PIPE)
```

Comment:
can you add a comment for this as 3.7 actually does the timing, where for < 3.7 asv is doing the timing.

Reply:
comment added + green